src/node_snapshotable.cc

```diff
@@ -883,8 +888,11 @@ bool SnapshotData::FromBlob(SnapshotData* out, FILE* in) {
   std::vector<char> sink(size);
   size_t num_read = fread(sink.data(), size, 1, in);
   CHECK_EQ(num_read, 1);
+  return FromBlob(out, sink);
```

Comment:
If we wrap the lines above this into a `FILE* -> std::vector` helper, then I think `EmbedderSnapshotData::FromFile()` can be implemented with just that helper and `EmbedderSnapshotData::FromBlob()`?

Reply:
Also done in 5fa6dec34dbf059d885b074f1953943a65c74f43 :+1: